### PR TITLE
Applied small changes to make stylish-haskell able to parse all sources

### PR DESCRIPTION
**Pull request by zeme-iohk**
stylish-haskell could not parse two files, but now it can.

### CODE REVIEW

File: nix/pkgs/fix-stylish-haskell/default.nix [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
-  # stylish-haskell can fail for some files that it can't parse, and fd will terminate
-  # if any of the exec commands fail. Rather than blacklisting the bad files, we simply
-  # ignore the exit code, which is usually unhelpful.
   ${fd}/bin/fd \
     --extension hs \
     --exclude 'dist-newstyle/*' \
     --exclude 'dist/*' \
     --exclude '.stack-work/*' \
-    --exec bash -c "${stylish-haskell}/bin/stylish-haskell -i {} || true"
+    --exec bash -c "${stylish-haskell}/bin/stylish-haskell -i {}"
```
Comment (zeme-iohk) [inline nix/pkgs/fix-stylish-haskell/default.nix:9]: This no longer applies as stylish-haskell never fails.

File: plutus-core/untyped-plutus-core/test/Evaluation/ApplyBuiltinName.hs [hunk 2]
```diff
@@ -48,7 +48,7 @@ type AppErr =
 newtype AppM a = AppM
     { unAppM :: Either AppErr a
     } deriving newtype (Functor, Applicative, Monad, MonadError AppErr)
-      deriving stock (MonadEmitter) via (NoEmitterT AppM)
+    deriving (MonadEmitter) via (NoEmitterT AppM)
```
Comment (zeme-iohk) [inline plutus-core/untyped-plutus-core/test/Evaluation/ApplyBuiltinName.hs:51]: stylish-haskell was unable to parse this.
This suggests that `deriving stock X via Y` is wrong, even though GHC doesn't complain.
I've built both version and the two resulting `ApplyBuiltinName.o` files are bit-identical

File: prettyprinter-configurable/Setup.hs [hunk 3]
```diff
@@ -7,18 +7,20 @@ module Main where
 #define MIN_VERSION_cabal_doctest(x,y,z) 0
 #endif
 
-#if MIN_VERSION_cabal_doctest(1,0,0)
+-- stylish-haskell is unable to parse import lines that are interspersed with
+-- other code, hence why we resort to using two #ifs instead of one.
```
Comment (zeme-iohk) [inline prettyprinter-configurable/Setup.hs:11]: As per the comment, this workaround is necessary to please stylish-haskell

Comment (michaelpj) [reply prettyprinter-configurable/Setup.hs:11]: Nice workaround!

**Review by michaelpj — APPROVED** [review]

**Pull request merged**